spec/controllers/curation_concern/generic_works_controller_spec.rb

```diff
@@ -44,12 +43,23 @@
   end
 
   describe "#create" do
+    let(:jill) { FactoryGirl.find_or_create(:jill) }
     it "should create a work" do
       expect {
         post :create, generic_work: { title: ["a title"] }
       }.to change { GenericWork.count }.by(1)
       expect(response).to redirect_to Sufia::Engine.routes.url_helpers.generic_work_path(assigns[:curation_concern])
     end
+
+    it "should record on_behalf_of" do
+      # The receiving user needs to have been persisted
+      jill.save 
+      post :create, generic_work: { id: 'test123', title: ["work title"], on_behalf_of: 'jilluser@example.com'}
+      expect(response).to redirect_to Sufia::Engine.routes.url_helpers.generic_work_path('test123')
+      saved_work = GenericWork.find('test123')
+      expect(saved_work.on_behalf_of).to eq 'jilluser@example.com'
+    end
+
```

Comment:
Having to do jill.save seems incorrect to me.  However, the receiving user needs to be extant (and persisted) in order for the transfer to function and the test to pass.  Is there a better way of doing this?